### PR TITLE
chore: added retrieveDepartmentByCode GW, contract test, added GW to …

### PR DESCRIPTION
**Pull request by nax2uk**
…appContainer and appContainer test

# What

- converted findWardByCode to retrieveDepartmentByCode GW
- added contract test for GW

# Why
mssql migrations
# Screenshots

# Notes

### CODE REVIEW

File: contractTest/gateways/MsSQL/retrieveDepartmentByCode.contractTest.js
Comment (faissalMT) [inline contractTest/gateways/MsSQL/retrieveDepartmentByCode.contractTest.js:33]: This line is unclear, it'd be fine to make use of a lookup table for this value such as what I've done here: https://github.com/madetech/nhs-virtual-visit/pull/803/files#diff-75191d314c7ca43884e00799166d15af75d459019461dea0e93713837c0f138aR12-R27

File: contractTest/gateways/MsSQL/retrieveDepartmentByCode.contractTest.js
Comment (faissalMT) [inline contractTest/gateways/MsSQL/retrieveDepartmentByCode.contractTest.js:9]: You won't need to do this once https://github.com/madetech/nhs-virtual-visit/pull/803 is merged, `setupOrganisationFacilityDepartmentAndManager()` will be able to create the Manager fine without the email needing to be specified because the database will be cleared between tests.

Comment (nax2uk) [reply contractTest/gateways/MsSQL/retrieveDepartmentByCode.contractTest.js:9]: Yes, this is good news! I will wait until #803 is merged.

File: contractTest/gateways/MsSQL/retrieveDepartmentByCode.contractTest.js
Comment (nax2uk) [reply contractTest/gateways/MsSQL/retrieveDepartmentByCode.contractTest.js:33]: Good idea, please have a look at my pm to you, for further suggestions for this lookup table. Let me know what you think.

Comment (nax2uk) [reply contractTest/gateways/MsSQL/retrieveDepartmentByCode.contractTest.js]: This has been done

File: contractTest/gateways/MsSQL/retrieveDepartmentByCode.contractTest.js
Comment (nax2uk) [reply contractTest/gateways/MsSQL/retrieveDepartmentByCode.contractTest.js]: This has been refactored for all previous contract test as well.

**Review by faissalMT — APPROVED** [review]

**Pull request merged**
